### PR TITLE
Add guarding in job parameters

### PR DESCRIPTION
**Pull request by BrunoChauvet**

### CODE REVIEW

File: app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb [hunk 1]
```diff
@@ -13,13 +13,23 @@ def enqueued?(organization_id)
     def find_job(organization_id)
       queue = Sidekiq::Queue.new(:default)
       queue.find do |job|
-        organization_id == job.item['args'][0]['arguments'].first
+        job_organization_id = begin
+          job.item['args'][0]['arguments'].first
+        rescue
+          job_organization_id
+        end
+        organization_id == job_organization_id
```
Comment (manu-d) [inline app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb]: organization_id = job.item['args'][0]['arguments'].first rescue nil

File: app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb [hunk 2]
```diff
@@ -13,13 +13,23 @@ def enqueued?(organization_id)
     def find_job(organization_id)
       queue = Sidekiq::Queue.new(:default)
       queue.find do |job|
-        organization_id == job.item['args'][0]['arguments'].first
```
Comment (manu-d) [inline app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb]:
```ruby
organization_id == job.item['args'][0]['arguments'].first rescue false
```
?

File: app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb [hunk 3]
```diff
@@ -13,13 +13,23 @@ def enqueued?(organization_id)
...
+        rescue
+          job_organization_id
+        end
+        organization_id == job_organization_id
       end
     end
 
     def find_running_job(organization_id)
       Sidekiq::Workers.new.find do |_, _, work|
-        work['queue'] == 'default' && work['payload']['args'][0]['arguments'].first == organization_id
```
Comment (manu-d) [inline app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb]:
```ruby
work['queue'] == 'default' && work['payload']['args'][0]['arguments'].first == organization_id rescue false
```
?

File: app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb (hunk 2 above)
Comment (BrunoChauvet) [reply app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb]: it's what I did initially but rubocop does not like inline rescue

**Pull request merged**